old/claude_processor-v1.py

```
@@ -0,0 +1,197 @@
+        
+        try:
+            parsed_data = json.loads(cleaned_json)
+            
+            # Basic validation of required fields
+            required_fields = ["booking_reference", "flights", "passengers"]
+            missing_fields = [field for field in required_fields if field not in parsed_data]
+            
+            if missing_fields:
+                print(f"Warning: Missing required fields: {', '.join(missing_fields)}")
+            
+            return parsed_data
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)